src/Stryker.CLI/Stryker.CLI/JsonConfigHandler.cs

```diff
@@ -47,6 +47,7 @@ public static void DeserializeConfig(string configFilePath, IStrykerInputs input
             inputs.LanguageVersionInput.SuppliedInput = jsonConfig.LanguageVersion;
             inputs.TestProjectsInput.SuppliedInput = jsonConfig.TestProjects;
             inputs.ExcludedMutationsInput.SuppliedInput = jsonConfig.IgnoreMutations;
+            inputs.ExcludedLinqExpressionInput.SuppliedInput = jsonConfig.IgnoreMutations;
```

Comment:
I changed to be only one input,  but keeped the separeted filter,  because  we starting discussing new rules with mutation level about which linq expressions should run, and having this filter separeted will help to implement this feature.

#1546

Reply:
That's a good thought but:

If you use a filter that means the mutation is still placed. The existing mutation level mechanism skips placing the mutation. See: https://github.com/stryker-mutator/stryker-net/blob/master/src/Stryker.Core/Stryker.Core/Mutators/MutatorBase.cs

I think it's better to use the existing mutation level mechanism for linq mutation levels and keep that separate from the filter.

Reply:
Hmm yeah the LinqMutator is the best place to make the logic for which expressions will run based on mutation level. ( I will create another issue for this one, to focus on this explicit configuration for the linq mutations)

So we keep the separated filter for the "ignore-mutations" configuration?

Reply:
~Well I don't think the filter really has anything linq specific so to me it makes the most sense to just fold it into the excluded mutations completely.~

I only just now realized linq methods have their own syntaxkind which makes this a bit different from exclude methods and I now see why you made it it's own filter.

I guess this is fine for now, though I think we should find a more generic way to filter specific mutations in the future so we can support other mutations.